CHANGELOG.md

```diff
@@ -25,6 +25,23 @@
 - On NetBSD, fixed crash due to incorrect detection of the main thread.
 - **Breaking:** The virtual key code `Subtract` has been renamed to `NumpadSubtract`
 - **Breaking:** On X11, `-` key is mapped to the `Minus` virtual key code, instead of `Subtract`
+- On Wayland, add missing virtual keycodes.
+- On Wayland, implement proper `set_cursor_grab`.
+- On Wayland, the cursor will use similar icons if the requested one isn't available.
+- On Wayland, right clicking on client side decorations will request application menu.
+- On Wayland, fix window size tracking between fullscreen state changes.
+- On Wayland, fix client side decorations not being hidden properly in fullscreen.
+- On Wayland, fix incorrect size event when entering fullscreen with client side decorations.
+- On Wayland, fix `resizable` attribute not being applied properly on startup.
+- On Wayland, fix disabled repeat rate not being handled.
+- On Wayland, fix decoration buttons not working after tty switch.
+- On Wayland, fix scaling not being applied on output re-enable.
+- On Wayland, fix crash when `XCURSOR_SIZE` is `0`.
+- On Wayland, fix pointer getting created in some cases without pointer capability.
+- On Wayland, on kwin, fix space between window and decorations on startup.
+- Added `title_font` and `title_color` methods on a `Theme` trait.
+- On Wayland, disable maximize button for non-resizable window.
+- On Wayland, fix window size tracking between tiled state changes.
```

Comment:
Could potentially be combined with ` - On Wayland, fix window size tracking between fullscreen state changes.`.

Something like this:

```
- On Wayland, fix tracking of window size after state changes.
```

Just an idea to reduce things a little bit, not sure how specific we need to be here.